### PR TITLE
Add RabbitMQ and Redis docker VMs

### PR DESCRIPTION
**Pull request by hodgestar**
Might as well add these and figure out how to hook them up to the other containers.

### CODE REVIEW

**Comment by hodgestar** [comment]
@smn @rudi - ready for review.

File: docker-vms/vxfreeswitch/vxfreeswitch.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ worker_name: vxfreeswitch_transport
 transport_name: vxfreeswitch_transport
 
 twisted_endpoint: "tcp:port=8000"
+twisted_client_endpoint: "tcp:host=prd-vumi-voice.za.prk-host.net:port=8021"
```
Comment (rudigiesler) [inline docker-vms/vxfreeswitch/vxfreeswitch.yaml]: Wasn't this only with the older version of vxfreeswitch (before your PR)?

**Comment by rudigiesler** [comment]
Just the one comment from me.

File: docker-vms/vxfreeswitch/vxfreeswitch.yaml (hunk 1 above)
Comment (hodgestar) [inline docker-vms/vxfreeswitch/vxfreeswitch.yaml]: Yep, but I think we still need to do a release of the new one.

Comment (rudigiesler) [inline docker-vms/vxfreeswitch/vxfreeswitch.yaml]: So we should make sure that this config works with both the old and new versions of vxfreeswitch?

Comment (hodgestar) [inline docker-vms/vxfreeswitch/vxfreeswitch.yaml]: I'll just do a release quickly and then fix this.

**Comment by hodgestar** [comment]
@rudigiesler Ready for re-review.

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**